glide.lock

```diff
@@ -1,37 +1,37 @@
-hash: d088a750a1a81b463b0056b81edd3ce849b9bef0eb44861f67041e13b95bade0
-updated: 2016-07-06T16:33:39.979049165+02:00
+hash: bb06d4eb9b26ddb217ba46a9c1924a870e9abbf3259dabffd19fa4407b7d769c
+updated: 2016-10-17T14:22:17.272063593-07:00
 imports:
 - name: github.com/appc/spec
-  version: fd5e3d8f5d596c6fbd506e80b5e13ba8a97b704c
+  version: f329150233aa3ac6c83ab5ba02a2b5a8bd03d365
```

Comment:
glide doesn't seem to offer the ability to just update a single package. If there is a way to do this, I'm happy to just add the new package without touching anything else.